util/pingpong.c

```diff
@@ -1953,6 +1977,9 @@ void pp_parse_opts(struct ct_pingpong *ct, int op, char *optarg)
 		ct->opts.dst_port = parse_ulong(optarg, UINT16_MAX);
 		break;
 
+	case 'm':
+		ct->tagged = strncasecmp("msg", optarg, 4);
```

Comment:
The call this is in already sets various hints, like domain name, changes endpoint type, etc.  I think we're okay modifying the caps as well.  I'd rather have a single variable to check than maintaining two that mirror each other.